client/lib/plans/test/get-popular-plan-spec.js

```diff
@@ -0,0 +1,75 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
I'd prefer not to introduce more usage of chai `expect`. There are some efforts to migrate to jest [`expect`](https://jestjs.io/docs/en/expect.html). It's in scope, no import necessary.

Reply:
Thanks, @sirreal -- updated.

I appreciate you keeping me current!

Reply:
Thank _you_! ✨